src/tools/build-manifest/src/main.rs

```diff
@@ -492,7 +492,7 @@ impl Builder {
             format!("clippy-{}-{}.tar.gz", self.clippy_release, target)
         } else if component == "rustfmt" || component == "rustfmt-preview" {
             format!("rustfmt-{}-{}.tar.gz", self.rustfmt_release, target)
-        } else if component == "llvm_tools" {
```

Comment:
This typo has been there since the beginning. It seems that this was working before only because the component was named "llvm-tools" (cf. else branch).